packages/destination-actions/src/destinations/airship/setAttributes/index.ts

```diff
@@ -0,0 +1,43 @@
+import type { ActionDefinition } from '@segment/actions-core'
+import type { Settings } from '../generated-types'
+import type { Payload } from './generated-types'
+import { setAttribute } from '../utilities'
+
+const action: ActionDefinition<Settings, Payload> = {
+  title: 'Set Attributes',
+  defaultSubscription: 'type = "identify"',
+  description: '',
+  fields: {
+    user: {
+      label: 'Airship Named User ID',
+      description: 'The identifier assigned in Airship as the Named User',
+      type: 'string',
+      required: false,
+      default: {
+        '@path': '$.userId'
+      }
+    },
+    occurred: {
+      label: 'Occurred',
+      description: 'When the Trait was set',
+      type: 'datetime',
+      required: true,
+      default: {
+        '@path': '$.receivedAt'
```

Comment:
Probably best to map this to the timestamp field, not the receivedAt field. 

```suggestion
        '@path': '$.timestamp'
```